api-proposal.md

```diff
@@ -127,14 +194,16 @@ Return a list of all tags for autocomplete for search.
 
 ---
 
-### Authentication
+#### User
 
-Is the user authenticated?
+Returns information about the currently signed-in user.
 
-- GET `/api/v1/auth`
+- GET `/api/v1/user`
   ```
   {
+    isAuthenticated: bool //
```

Comment:
Do we need `isAuthenticated`, I just assumed this request would 40X or 50X if the user wasn't logged in.